scripts/runfabtests.sh

```diff
@@ -191,18 +239,19 @@ function unit_test {
 		return
 	fi
 
-	start_time=$(date '+%s.%N')
+	start_time=$(date '+%s')
 
-	${ssh} ${SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
+	${SSH_SERVER} ${BIN_PATH} "${test_exe}" &> $s_outp &
 	p1=$!
 
 	wait $p1
 	ret1=$?
 
-	end_time=$(date '+%s.%N')
+	end_time=$(date '+%s')
 	test_time=$(compute_duration "$start_time" "$end_time")
 
-	if [ $ret1 -eq 61 ]; then
+        local nr=$(not_run $ret1)
```

Comment:
@pmmccorm I agree, and let's fix it for v1.2.  In the meantime, I will make the brew test (which is written for 1.X, so as to get it in soon) very simple, with an explanation that for 1.2 it will be updated.